dashboard/app/views/courses/index.html.haml

```diff
@@ -15,6 +15,7 @@
   courses_data[:initialtipsdismissed] = current_user ? current_user.ui_tip_dismissed_teacher_courses : nil
   courses_data[:userid] = current_user.try(:id)
   courses_data[:modernelementarycoursesavailable] = @modern_elementary_courses_available
+  courses_data[:displayMinecraftAquatic] = ['mc', 'dance'].include? DCDO.get('hoc_launch', nil)
```

Comment:
Maybe `false` instead of `nil`?

Reply:
both `false` and `nil` will result in `displayMinecraftAquatic` being set to `false`, so i'm not sure i understand the value in making that change

Reply:
I guess it would be more important if `displayMinecraftAquatic` were `isRequired`, of course, at which point it can't be `null`.

Reply:
that's what i'm saying -- if `DCDO.get('hoc_launch', nil)` does return `nil`, `displayMinecraftAquatic` will still be `false` (not `null`) because `['mc', 'dance'].include?(nil) => false` and `.include?` always returns a boolean value

Reply:
(i totally realize this was just a nit but want to make sure i'm not super confused about something)

Reply:
Oh good call, you're totally right.